benchflow-dsl/src/main/scala/cloud/benchflow/dsl/definition/configuration/goal/GoalYamlProtocol.scala

```diff
@@ -18,9 +18,12 @@ import scala.util.Try
 object GoalYamlProtocol extends DefaultYamlProtocol {
 
   val TypeKey = YamlString("type")
+  val StoredKnowledgeKey = YamlString("stored_knowledge")
   val ObservationKey = YamlString("observation")
   val ExplorationSpaceKey = YamlString("exploration_space")
 
+  val StoredKnowledgeDefaultValue = true
```

Comment:
I added the following comment yesterday after our discussion: https://github.com/benchflow/benchflow/issues/437#issuecomment-309818624. Better to have it in an object, also for the other cases.